README.rst

```diff
@@ -0,0 +1,301 @@
+Intel® Media SDK
+================
+
+.. contents::
+
+Intel® Media SDK provides a plain C API to access hardware-accelerated video decode, encode and filtering on Intel® Gen graphics hardware platforms. Implementation written in C++ 11 with parts in C-for-Media (CM).
+
+**Supported video encoders**: HEVC, AVC, MPEG-2, JPEG, VP9  
+
+**Supported video decoders**: HEVC, AVC, VP8, VP9, MPEG-2, VC1, JPEG, AV1  
+
+**Supported video pre-processing filters**: Color Conversion, Deinterlace, Denoise, Resize, Rotate, Composition  
+
+Media SDK is a part of Intel software stack for graphics:
+
+* `Linux Graphics Drivers <https://intel.com/linux-graphics-drivers>`_ - General Purpose GPU Drivers for Linux&ast; Operating Systems
+
+  * Visit `documentation <https://dgpu-docs.intel.com>`_ for instructions on installing, deploying, and updating Intel software to enable general purpose GPU (GPGPU) capabilities for Linux&ast;-based operating system distributions.
+
+Media SDK Support Matrix
+------------------------
+
+Pay attention that Intel® Media SDK lifetime comes to an end in a form it
+exists right now. In particular, 
+
+* API 1.35 is projected to be the last API of 1.x API series
+* Runtime library (libmfxhw64.so.1) is not planned to get support of new Gen platforms
+* Project is going to be supported in maintainence mode, critical fixes only
+
+All future development is planned to happen within
+[oneVPL](https://github.com/oneapi-src/oneVPL) library and its runtime
+implementations which are direct successors of Intel® Media SDK. oneVPL introduces
+API 2.x series which is not backward compatible with API 1.x series (some
```

Comment:
Shall we add a link to https://software.intel.com/content/www/us/en/develop/articles/upgrading-from-msdk-to-onevpl.html . It has a list of these features.

Reply:
Is there a link to VPL spec instead? I guess it might have such information as well. I would prefer to refer to a more formal spec here, but we can also add a list of useful links for more reading in the end of whole "Matrix" paragraph.

Reply:
https://spec.oneapi.com/versions/latest/elements/oneVPL/source/index.html

Reply:
I think here it is:
* https://spec.oneapi.com/versions/latest/elements/oneVPL/source/appendix/VPL_intel_media_sdk.html

Reply:
It's a good link

Reply:
@aobolensk : can you, please, add the link here:
```
some `features got dropped <https://spec.oneapi.com/versions/latest/elements/oneVPL/source/appendix/VPL_intel_media_sdk.html>`_
```

Reply:
Done